debian_packages.yaml

```diff
@@ -1,12 +1,11 @@
-# debian 11
-- distros: ["debian11"]
-  archs: ["amd64", "arm64", "arm", "s390x", "ppc64le"]
```

Comment:
This doesn't seem right.

Plenty of folks us using `arm64` cpu. Also removing less common stuff like `power pc` without wide discussion seems to be wrong in my opinion. I'd bet maintainers would reject PR base on this change